File: app/src/main/java/ds/photosight/model/Comment.java
package ds.photosight.model;

import java.io.Serializable;
import java.util.Date;

public class Comment implements Serializable{

	public String content;
	public String name;
	public String status;
	public String avatarUrl;
	public String rate;
	public String dateRaw;
	public String dateFormatted;
	public Date date;

}

File: app/src/main/java/ds/photosight/model/Comment.kt
package ds.photosight.model

import java.io.Serializable
import java.util.Date

public data class Comment(
        var content: String? = null,
        var name: String? = null,
        var status: String? = null,
        var avatarUrl: String? = null,
        var rate: String? = null,
        var dateRaw: String? = null,
        var dateFormatted: String? = null,
        var date: Date? = null
) : Serializable {


}
